backend/service/service_proposal.go

```diff
@@ -223,18 +378,25 @@ func (service *ProposalService) QueryList(page, size int, istotal bool) (resp vo
 		}
 
 		if propo.Status == document.ProposalStatusVoting {
+			curValidators := make(map[string]*vo.ValidatorGovInfo, len(propo.Votes))
+			curDelegators := make(map[string]*vo.DelegatorGovInfo, len(propo.Votes))
 
 			for _, v := range propo.Votes {
-				var voterVotingPower int64
-				voterInfo := addrAsMultiTypeMap[v.Voter]
+				getDelegationsByVoter(curValidators, curDelegators, v, allBondedValidators)
```

Comment:
set value of `curValidators`  before loop `propo.Votes`

Reply:
ok